nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/SchemaManagerImpl.java

```diff
@@ -550,11 +550,16 @@ protected DocumentType recomputeDocumentType(String name, DocumentTypeDescriptor
         Set<String> facetNames = new HashSet<>();
         Set<String> schemaNames = SchemaDescriptor.getSchemaNames(dtd.schemas);
         facetNames.addAll(Arrays.asList(dtd.facets));
+        Set<String> subtypes = new HashSet<>(Arrays.asList(dtd.subtypes));
+        Set<String> blacklisted = new HashSet<>(Arrays.asList(dtd.blacklistedSubtypes));
 
         // inherited
         if (parent != null) {
             facetNames.addAll(parent.getFacets());
             schemaNames.addAll(Arrays.asList(parent.getSchemaNames()));
+            // not inheriting subtypes and blacklisted subtypes for compatibility with platform types
+            //subtypes.addAll(parent.getSubtypes());
```

Comment:
Why do we keep that as comments as it won't be uncommented anyway (according to the first comment)?